contributor_docs/inline_documentation.md

```diff
@@ -226,12 +229,10 @@ function setup() {
 
 If you need to link to external asset files, put them in [/docs/yuidoc-p5-theme/assets](https://github.com/processing/p5.js/tree/master/docs/yuidoc-p5-theme/assets) and then link to them with "assets/filename.ext" in the code. See the [tint example](http://p5js.org/reference/#/p5/tint).
 
-On some code editors copy-pasting examples in the inline documentation may be awkward and require a lot of adjustments (tabs, asterisks, spaces). You can use this simple tool to beautify the code (and re-indent it to 2 spaces) and make it ready for a copy-paste: 
-[p5examplesFormatter](http://molleindustria.org/p5examplesFormatter/)
-| [download as zip](http://molleindustria.org/p5examplesFormatter/p5examplesFormatter.zip)
+On some code editors copy-pasting examples in the inline documentation may be awkward and require a lot of adjustments (tabs, asterisks, spaces).
```

Comment:
I only removed the link because it is no longer alive. It just created confusion. But kept the paragraph as a "watch-out" note for a potential problem.

Reply:
It's better to have some sort of advice to mitigate the problem of "copy-pasting examples in the inline documentation may be awkward and require a lot of adjustments" otherwise it could create confusion about what people writing examples should do about this problem.

Reply:
Do we have any another solution to this problem ? Otherwise i am fine with completely removing it. Shall i do it ?

Reply:
Yes,i agree some solution would be nice. But i am not sure what that solution can be. Can we get a copy of the beautifier tool, so that we can simply host it on the repo and just update the link ?

Reply:
I think we can remove this paragraph as the template should give a good enough idea about what to do in terms of formatting them. Adding the asterisks is a bit troublesome but not too much so and a modern text editor should be able to handle that well enough.

I'm not sure what the beautifier tool is, never used it and this really is the first time I've heard of it.

Reply:
let's remove this whole paragraph